wrappers/python/examples/box_dimensioner_multicam/realsense_device_manager.py

```diff
@@ -113,48 +117,64 @@ def post_process_depth_frame(depth_frame, decimation_magnitude=1.0, spatial_magn
 
 
 class DeviceManager:
-    def __init__(self, context, pipeline_configuration):
+    def __init__(self, context, D400_pipeline_configuration, L500_pipeline_configuration = rs.config()):
         """
         Class to manage the Intel RealSense devices
 
         Parameters:
         -----------
         context                 : rs.context()
                                   The context created for using the realsense library
-        pipeline_configuration  : rs.config()
-                                  The realsense library configuration to be used for the application
+        D400_pipeline_configuration  : rs.config()
+                                  The realsense library configuration to be used for the application when D400 product is attached.
+                                  Pass "rs.config()" if no D400 product attached
```

Comment:
Remove line -- as I understand it, it always works...

Reply:
done